053-SentinelAutomatedResponse/Coach/README.md

```diff
@@ -0,0 +1,68 @@
+
+<!-- REPLACE_ME (this section will be removed by the automation script) -->
+# What The Hack - Sentinel Automated Response - Coach Guide
+<!-- REPLACE_ME (this section will be removed by the automation script) -->
+
+## Introduction
+
+<!-- REPLACE_ME (this section will be removed by the automation script) -->
+Welcome to the coach's guide for the Sentinel Automated Response What The Hack. Here you will find links to specific guidance for coaches for each of the challenges.
+<!-- REPLACE_ME (this section will be removed by the automation script) -->
+
+**NOTE:** If you are a Hackathon participant, this is the answer guide. Don't cheat yourself by looking at these during the hack! Go learn something. :)
+
+## Coach's Guides
+
+This hack includes optional [lecture presentations](../Coach/Resources) that introduces key topics associated with each challenge. It is recommended that the host present each short presentation before attendees kick off that challenge.</br>
+
+<!-- REPLACE_ME (this section will be removed by the automation script) -->
+- Challenge-01: **[Architecture, Agents, Data Connectors and Workbooks](Solution-01.md)**
+   - Understand the various architecture and decide on the appropriate design based on the requirements. Install the appropriate data connector to import Windows security events and validate Log Analytics data ingestion.
+- Challenge-02: **[Custom Queries & Watchlists](Solution-02.md)**
+   -  Build a custom analytics rule to show data ingested through the connector. Create a Watchlist and add data, verify the data is available in Log Analytics.  Change the table retention time to 7 days
+- Challenge-03: **[Automated Response](Solution-03.md)**
+   -  Bulid a custom rule that alerts when your user ID logs into a server. Use a playbook to automatically close the incident only if login occurred from a known IP address
+<!-- REPLACE_ME (this section will be removed by the automation script) -->
+
+
+## Prerequisites
+
+- Access to the Internet - Azure Docs
+- Azure subscription with contributor role
+- Prebuild a generic Windows server running in the subscription.
+
+</br>
+
+## Student Resources
+
+Before the hack, it is the Coach's responsibility to download and package up the contents of the \`/Student/Resources\` folder of this hack into a "Resources.zip" file. The coach should then provide a copy of the Resources.zip file to all students at the start of the hack.
```

Comment:
Since your hack is not providing students with any resource files, you may remove this paragraph from the boilerplate tempate.  However, leave the next two paragraphs in about the WTH website and the NOTE to never share the repo link with students.